docs/en/install_deploy.md

```diff
@@ -0,0 +1,393 @@
+
+```
+wget https://archive.apache.org/dist/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz
+cd zookeeper-3.4.14
+cp conf/zoo_sample.cfg conf/zoo.cfg
+```
+
+#### 2. Modify the Configuration File
+
+Open the file `conf/zoo.cfg` and modify `dataDir` and `clientPort`.
+
+```
+dataDir=./data
+clientPort=7181
+```
+
+#### 3. Start Zookeeper
+
+```
+sh bin/zkServer.sh start
+```
+
+Deploy the Zookeeper cluster [refer to here](https://zookeeper.apache.org/doc/r3.4.14/zookeeperStarted.html#sc_RunningReplicatedZooKeeper).
+
+### Deploy tablet
+
+#### 1 Download the OpenMLDB deployment package
+
+```
+wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.4.3/openmldb-0.4.3-linux.tar.gz
+tar -zxvf openmldb-0.4.3-linux.tar.gz
+mv openmldb-0.4.3-linux openmldb-tablet-0.4.3
+cd openmldb-tablet-0.4.3
+```
+
+#### 2 Modify the Configuration File: conf/tablet.flags
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+* Modify `zk_cluster` to the already started zk cluster address.
+* If you share zk with other OpenMLDB, you need to modify `zk_root_path`.
+
+```
+--endpoint=172.27.128.33:9527
+--role=tablet
+
+# If tablet run as cluster mode zk_cluster and zk_root_path should be set:
+--zk_cluster=172.27.128.33:7181,172.27.128.32:7181,172.27.128.31:7181
+--zk_root_path=/openmldb_cluster
+```
+
+**Notice:**
+* The endpoint cannot use 0.0.0.0 and 127.0.0.1.
+* If the domain name is used here, all the machines where the client using openmldb is located must be equipped with the corresponding host. Otherwise, it will not be accessible.
+* The configuration of zk_cluster and zk_root_path is consistent with that of Nameserver.
+
+#### 3 Start the service
+
+```
+sh bin/start.sh start tablet
+```
+
+Repeat the above steps to deploy multiple tablet.
+
+**Notice:**
+* After the service is started, the tablet.pid file will be generated in the bin directory, and the process number at startup will be saved in it. If the pid inside the file is running, the startup will fail.
+* Cluster version needs to deploy at least 2 tablets.
+* If you need to deploy multiple tablets, deploy all the tablets before deploying the Nameserver.
+
+### Deploy Nameserver
+
+#### 1 Download the OpenMLDB deployment package
+
+```
+wget https://github.com/4paradigm/OpenMLDB/releases/download/v0.4.3/openmldb-0.4.3-linux.tar.gz
+tar -zxvf openmldb-0.4.3-linux.tar.gz
+mv openmldb-0.4.3-linux openmldb-ns-0.4.3
+cd openmldb-ns-0.4.3
+```
+
+#### 2 Modify the Configuration File: conf/nameserver.flags
+
+* Modify `endpoint`. The endpoint is the deployment machine ip/domain name and port number separated by colons.
+* Modify `zk_cluster` to the address of the zk cluster that has been started. Ip is the ip of the machine where zk is located, and port is the port number configured by clientPort in the zk configuration file. If zk is in cluster mode, separate it with commas, and the format is ip1:port1,ip2:port2, ip3:port3.
```

Comment:
Ip is the ip of the machine where zk is located =>
IP is the machine address where zk is located,

Reply:
Ok, thank you.